tests/integration/cmd_link_unlink_test.go

```diff
@@ -9,138 +13,199 @@ import (
 
 var _ = Describe("odo link and unlink command tests", func() {
 
-	/*
-		Uncomment when we uncomment the test specs
-		var frontendContext, backendContext string
-		var oc helper.OcRunner
-	*/
-
 	var commonVar helper.CommonVar
 
-	// This is run before every Spec (It)
 	var _ = BeforeEach(func() {
-		// oc = helper.NewOcRunner("oc")
 		commonVar = helper.CommonBeforeEach()
+		// wait till odo can see that all operators installed by setup script in the namespace
+		odoArgs := []string{"catalog", "list", "services"}
+		operator := "service-binding-operator"
+		helper.WaitForCmdOut("odo", odoArgs, 5, true, func(output string) bool {
+			return strings.Contains(output, operator)
+		})
 	})
 
-	// Clean up after the test
-	// This is run after every Spec (It)
 	var _ = AfterEach(func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
-	Context("when running help for link and unlink command", func() {
-		It("should display the help", func() {
+	Context("Running the help command", func() {
+		It("should display help for the link command", func() {
 			appHelp := helper.Cmd("odo", "link", "-h").ShouldPass().Out()
-			helper.MatchAllInOutput(appHelp, []string{"Link component to a service ", "backed by an Operator or Service Catalog", "or component", "works only with s2i components"})
-			appHelp = helper.Cmd("odo", "unlink", "-h").ShouldPass().Out()
+			helper.MatchAllInOutput(appHelp, []string{"Link component to a service ", "backed by an Operator or Service Catalog", "or component"})
+		})
+		It("should display help for the unlink command", func() {
```

Comment:
Why are we spitting out help commands? I think this way it will increase the test execution time. I mean just for testing the help commands we need to run `BeforeEach` part twice in this case. Don't you think just for `help` command it costs much. Correct me if I am wrong.

Reply:
I agree with what you say, and I contemplated it for a while; whether to go with the test design or with saving the test execution. I got carried away with the design :sweat_smile: